src/uct/tcp/tcp_iface.c

```diff
@@ -591,9 +591,10 @@ static ucs_status_t uct_tcp_iface_listener_init(uct_tcp_iface_t *iface)
         goto err_close_sock;
     }
 
-    ucs_debug("tcp_iface %p: listening for connections (fd=%d) on %s",
-              iface, iface->listen_fd, ucs_sockaddr_str((struct sockaddr *)&bind_addr,
-                                                       ip_port_str, sizeof(ip_port_str)));
+    ucs_debug("tcp_iface %p on dev %s: listening for connections (fd=%d) on %s",
```

Comment:
ucs_debug("tcp_iface %p: listening for connections (fd=%d) on %s netif %s",

Reply:
Done